po/nb.po

```diff
@@ -0,0 +1,115 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the io.github.zefr0x.hashes package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: io.github.zefr0x.hashes\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-02-29 23:09+0300\n"
+"PO-Revision-Date: 2024-07-12 17:17+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: nb\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.4.4\n"
+
+#: data/io.github.zefr0x.hashes.desktop.in:5
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:6 hashes/ui/main_window.py:29
+msgid "Hashes"
+msgstr "Hashes"
+
+#: data/io.github.zefr0x.hashes.desktop.in:6
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:7
+msgid "Identify hashing algorithms"
+msgstr "Avslør avtrykksalgoritmer"
+
+#: data/io.github.zefr0x.hashes.desktop.in:7
+msgid "Hash Identifier"
+msgstr "Avslør avtrykksalgoritmer"
+
+#: data/io.github.zefr0x.hashes.desktop.in:13
+msgid "hash;hashcat;john;detect;identify"
+msgstr "hash;hashcat;john;gjenkjenning;identifisering"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:9
+msgid ""
+"This utility lets you conveniently find out what hashing algorithm has been "
+"used on a particular string."
+msgstr ""
+"Med dette verktøyet kan den indre detektiven i deg finne ut hvilken "
+"avtrykksalgoritme som har blitt brukt på et gitt stykke tekst."
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:10
+msgid "Features:"
+msgstr "Her er noen ting du vil sette ekstra pris på med Hashes:"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:12
+msgid "📺 Popularity Ratings - Most popular hashes showed first."
+msgstr "Populære algoritmer vises øverst."
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:13
+msgid ""
+"👵 Updated! - A new hash algorithm database is used in the identification "
+"process."
+msgstr "Appen bruker en moderne database med de nyeste algoritmene."
```

Comment:
I liked how you translated this part. However, wouldn't it be more friendly and easy to understand with those emojis? Or is there a cultural reason for not including them?

Reply:
It's pretty common to use emojis in English software descriptions, but in a Norwegian cultural context it comes off as slightly over-the-top and exaggerated. So yeah, it's a mostly a cultural thing, I suppose. We're a reserved people :)